inc/class-wpseo-utils.php

```diff
@@ -483,35 +492,266 @@ public static function clear_transient_cache( $option ) {
 	 *
 	 * @param array $types Set of sitemap types to invalidate cache for.
 	 */
-	public static function clear_sitemap_cache( $types = array() ) {
-		global $wpdb;
+	public static function clear_sitemap_cache( array $types = array() ) {
+		// Filter out optional empty items.
+		$types = array_filter( $types );
```

Comment:
Cool, thought you were missing the callback, didn't know that PHP will delete all false values. :+1: